sites/public/src/components/listings/search/ListingsSearchCombined.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from "react"
+import { ListingSearchParams, generateSearchQuery } from "../../../lib/listings/search"
+import { ListingService } from "../../../lib/listings/listing-service"
+import { ListingsCombined } from "../ListingsCombined"
+import { AppearanceBorderType, AppearanceSizeType, Button } from "@bloom-housing/ui-components"
```

Comment:
Can you import these from doorway-ui-components instead to keep up with styling changes that have already been made?

Reply:
My understanding was that we would use the doorway UIC only for components that had been modified and use base UIC for everything else.  Are you saying that the `Button` component has been modified and that it should be used in place of the default?

Reply:
Yes, Button has been modified.

Reply:
Got it.  Thanks for letting me know.  I've update to the doorway version